client/asset/dcr/rpcwallet.go

```diff
@@ -525,6 +525,17 @@ func (w *rpcWallet) InternalAddress(ctx context.Context, acctName string) (stdad
 	return addr, translateRPCCancelErr(err)
 }
 
+// ExternalAddress returns an external address from the specified account using
+// GapPolicyWrap. The dcrwallet user should set --gaplimit= as needed to prevent
+// address reused depending on their needs. Part of the Wallet interface.
+func (w *rpcWallet) ExternalAddress(ctx context.Context, acctName string) (stdaddr.Address, error) {
+	addr, err := w.rpcClient.GetNewAddressGapPolicy(ctx, acctName, dcrwallet.GapPolicyWrap)
```

Comment:
As the revised comment states, with an external dcrwallet, the user can pick their `--gaplimit` as desired.  Using ignore with the RPC has created difficult recover situations.  The `InternalAddress` method already uses wrap implicitly (`getrawchangeaddress` does not even expose gap policy).

Reply:
When an address is used, the next (unused) address index becomes the address to which wrapping returns.  The wrap algo just ensures the gap limit isn't exceeded.